app/views/gettingStarted.scala.html

```diff
@@ -1,40 +1,109 @@
-
-            <hr />
-
-            <h3>Downloading and building examples</h3>
-            <p>Lightbend <a href="//developer.lightbend.com/start/?group=play" target="_blank" rel="noopener noreferrer">Tech Hub</a> offers a variety of Play examples for Java and Scala. We recommend trying the Play Starter Example first.</p> 
-            <p>The downloadable zip files include everything you need to build and run the examples, including a distribution of the <a href="//www.scala-sbt.org/index.html" target="_blank" rel="noopener noreferrer">sbt</a>. If you have <a href="//gradle.org/" target="_blank" rel="noopener noreferrer">Gradle</a>, you can use it as a build tool instead.</p>
-
-            <p>After choosing an example from <a href="//developer.lightbend.com/start/?group=play" target="_blank" rel="noopener noreferrer">Tech Hub</a> follow these steps:</p>
-            <ol>
-                <li>Click CREATE A PROJECT FOR ME to download the zipped project.</li>
-                <li>Unzip the project in a convenient location.</li>
-                <li>In a command window, navigate to the top level project directory.</li>
-                <li>Enter one of the following commands:
-                    <ul>
-                        <li>On OSx or Linux systems: <code>./sbt run</code> or <code>./gradle runPlayBinary</code></li>
-                        <li>On Windows systems: <code>sbt.bat run</code> or <code>gradlew.bat runPlayBinary</code></li>
-                    </ul>
-                    <p>The build tool downloads dependencies and compiles the project.</p>
-                </li>
-                <li>
-                    After the build finishes, enter the following URL in a browser to view the app:
-                    <p><a href="http://localhost:9000"><code>http://localhost:9000/</code></a></p>
-                </li>
-            </ol>
-            <p>Check out the readme file in the top level project directory to learn more about the example.</p>
+
+            <div class="try-section">
+                <h2>New to Play?</h2>
+                <p>Start with <a href="@controllers.documentation.ReverseRouter.latest(None)">our documentation</a> or the <a href="@controllers.documentation.ReverseRouter.latest(None, "HelloWorldTutorial")">Hello World Tutorial</a>.</p>
+
+                <div class="try-option">
+                    <h3><span>&raquo;</span> Learn whether Play Framework is right for your project</h3>
+                    <div class="try-option-content">
+                        <ul>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Introduction")">Introduction to Play</a></li>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")">Requirements to work With Play</a></li>
```

Comment:
Either `System Requirements` or `Requirements for using Play` read better to me. 😄